torch/_prims_common/wrappers.py

```diff
@@ -101,9 +104,11 @@ def __init__(
         *,
         type_promotion_kind: ELEMENTWISE_TYPE_PROMOTION_KIND,
         type_promoting_args: Optional[Sequence[str]] = None,
+        use_dtype_as_return_type: bool = True,
```

Comment:
The dtype behaviour is, or should be, consistent throughout PyToch.

Let's not add more arguments for the sake of adding them, things are already difficult enough as they are.

Reply:
removed